index.html

```diff
@@ -1,10 +1,105 @@
+                            <div class="utomi">
+                                <div class="search">
+                                    <input id="search-input" type="text">
+                                    <i class="fa fa-fw fa-search"></i>
+                                </div>
+                            </div>
+                        </li>
+
+                    </ul>
+
+
+                </nav>
+
+                <div class="nav-titles">
+                    <p class="big-text"><a href="#">What will your verse be?</a></p>
+                    <p class="small-text"> Everyone has something to share. See how people are using <br><span>iPad in
+                            their
+                            unique ways to add to the world story</span></p>
+                    <p class="smaller-text">Watch the "Your Verse" film and explore the stories <i
+                            class="fa fa-play-circle"></i></p>
+
+                </div>
+
+            </header>
+        </div>
+
+        <aside class="images-container">
+            <a  href="#"><img  src="images/iphone5s.jpg" alt="iphone products"></a>
+            <a  href="#"><img  src="images/image-b.jpg" alt="iphone products"></a>
+            <a  href="#"><img  src="images/image-c.jpg" alt="iphone products"></a>
+            <a  href="#"><img  src="images/image-d.jpg" alt="iphone products"></a>
+
+        </aside>
+
+        <footer>
+            <div class="footer-wrapper">
+                <div class="ft-line1">
+                    <div class="div-p-ul">
+                        <div>
+                            <p>Shop the Apple <a href="#">Online Store</a> (1-800-MY-APPLE), visit an <a href="#">Apple
+                                    Retail Store,</a> or find a <a href="#">reseller.</a>
+                            </p>
+                        </div>
+                        <div>
+                            <ul class="ul-footer">
+                                <li><a href="#">Site Map</a></li>
+                                <li><a href="#">Hot News</a></li>
+                                <li><a href="#">RSS Feeds</a></li>
+                                <li><a href="#">Media Info</a></li>
+                                <li><a href="#">Environment</a></li>
+                                <li><a href="#">Job Opportunities</a></li>
+                                <li><a href="#">Contact Us</a></li>
+                                <li><a href="#" class="no-border"><img src="images/flag-400.png" alt="USA flag"></a>
+                                </li>
+                            </ul>
+                        </div>
+                    </div>
+                </div>
+
+                <div class=ft-line-2>
+                    <P>Copyright © 2014 Apple Inc. All rights reserved. <span><a href="#">Terms of Use</a></span>
+                        |<span><a href="#">Privacy
+                                Policy</a></span></P>
+                </div>
```

Comment:
You should place this box inside of the box that contains the first paragraph, it will be easier for you to style it after that.